esp/src/CMakeLists.txt

```diff
@@ -45,8 +45,8 @@ else ()
                 set (NODE_ERROR \"Unable to locate node/npm, please install NodeJS as per https://github.com/hpcc-systems/HPCC-Platform/wiki/Building-HPCC#prerequisites\" )
             endif ()
 
-            if ( NOT \"${NODE_ERROR}\" STREQUAL \"\")
-                message ( WARNING \"${NODE_ERROR}\" )
+            if ( NOT \"\${NODE_ERROR}\" STREQUAL \"\")
```

Comment:
I don't understand why the " and the $ need escaping here, but if they do then they presumably do a couple of lines later too...

Reply:
Actually I think I do understand, and it's probably ok in the other places (will just expand earlier, but that's ok for the other cases).